lib/src/slices/api_microsoft/api_microsoft_service.dart

```diff
@@ -1,37 +1,35 @@
+import '../api_auth_service/api_auth_service.dart';
 import '../api_auth_service/model/api_auth_service_account_model.dart';
-import '../api_auth_service/model/api_auth_service_rsp.dart';
-import '../api_email_msg/model/api_email_msg_model.dart';
-import '../data_bkg/api_email_provider_abstract.dart';
+import '../data_bkg/data_bkg_service_provider.dart';
+
+class ApiMicrosoftService implements DataBkgServiceProviderInterface {
+  final ApiAuthServiceAccountModel _account;
```

Comment:
the accounts are stored separately, they don't need to be in here otherwise you'll be creating a service per account